### DISCUSSION TITLE
How to return default value in getMany if null

### DISCUSSION THREAD

**Discussion by wh1t3h47** [prisma/prisma#15462] [Q&A] [answered]
I have a query that works like this:
```typescript
const fruits = await this.fruitsRepository.findMany({
      select: {
        fruitProperties: {
          color: true,
          taste: true,
        }
      },
    });
```

Supposing fruitProperties is an array (one fruit has multiple fruitProperties), how can I return another value if fruitProperties is null in fruits table? Sql has IFNULL() function

**Comment by nurul3101** [comment, marked as answer]
Hi @wh1t3h47 👋 

For this case you would need to use [Raw Query](https://www.prisma.io/docs/concepts/components/prisma-client/raw-database-access#raw-queries-with-relational-databases), You can pass the query string having IFNULL() function into $queryRaw.

**Reply by wh1t3h47** [reply]
Thank you very much :D

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
